src/tracing/swapFlowLoggers.ts

```diff
@@ -32,3 +33,25 @@ export function maybeLogFirstSwapAction(analyticsContext: ITraceContext) {
     })
   }
 }
+
+export function logSwapQuoteRequest(
+  chainId: number,
+  routerPreference: RouterPreference | typeof INTERNAL_ROUTER_PREFERENCE_PRICE
+) {
+  let performanceMetrics = {}
+  if (routerPreference !== INTERNAL_ROUTER_PREFERENCE_PRICE) {
```

Comment:
added since last commit - we don't want to log the performance metrics for a random price quote